src/main/java/frc/robot/drivetrain/Module.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot.drivetrain;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModulePosition;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.math.util.Units;
+import org.littletonrobotics.junction.Logger;
+
+public class Module {
+
+  public static final double kDriveRatio = 6.25;
```

Comment:
Can you add a comment describing where this value came from? My guess was from the SDS module but their listed value is 6.12

![image](https://github.com/team4909/2024-Crescendo/assets/175873/8a5bf591-9174-4231-a7fe-2a2ee820161f)